templates/strapi/files/config/environments/production/database.js

```diff
@@ -0,0 +1,36 @@
+const config = require("platformsh-config").config();
```

Comment:
Does Strapi not have an "All environments" directory for config the way Symfony and Laravel do?